modules/ROOT/pages/anypoint-platform-cli-commands.adoc

```diff
@@ -1499,7 +1503,74 @@ include::partial$cli-default-options.adoc[tag=CLIdefaultOptions]
 This command describes the asset passed in `<assetIdentifier>`.
 
 Argument `assetIdentifier` should be formatted as follows: `([group_id]/)<asset_id>/<version>`. +
-If `group_id` is not specified, it defaults to the currently selected Organization ID. 
+If `group_id` is not specified, it defaults to the currently selected Organization ID.
+
+// CLI DEFAULTS
+include::partial$cli-default-options.adoc[tag=CLIdefaultOptions]
```

Comment:
👀 
![image](https://user-images.githubusercontent.com/62706868/125303296-14e6a900-e303-11eb-8e74-99565e216526.png)

Reply:
🤷‍♂️ all doc content has the same link to default optional arguments